x/stakeibc/types/message_rebalance_validators_test.go

```diff
@@ -27,6 +32,27 @@ func TestMsgRebalanceValidators_ValidateBasic(t *testing.T) {
 				Creator: sample.AccAddress(),
 			},
 			err: sdkerrors.ErrInvalidAddress,
+		}, {
```

Comment:
@sampocs do you think we should move these tests back to `x/stakeibc/keeper/msg_server_rebalance_validators_test.go` so that we have consistency in our testing approach across the codebase?